.azure/pipelines/richnav.yml

```diff
@@ -6,58 +6,68 @@
 trigger:
   branches:
     include:
-    - blazor-wasm
     - main
     - release/*
-    - internal/release/*
+
+# Do not run this pipeline for PR validation.
+pr: none
 
 variables:
 - name: _BuildArgs
   value: '/p:SkipTestBuild=true'
```

Comment:
Thoughts on whether turning off rich navigation for test code makes sense anyone❔